src/modules/LanItems.rb

```diff
@@ -2667,125 +2665,64 @@ def export_udevs(devices)
           )
         end
         Builtins.foreach(devs) do |device|
-          driver = Convert.convert(
-            SCR.Execute(
-              path(".target.bash_output"),
-              Builtins.sformat(
-                "driver=$(ls -l /sys/class/net/%1/device/driver);echo ${driver##*/}|tr -d '\n'",
-                device
-              )
-            ),
-            from: "any",
-            to:   "map <string, any>"
-          )
+          begin
+            driver = begin
+                       File.readlink("/sys/class/net/#{device}/device/driver")
+                     rescue
+                       nil
+                     end
+          rescue Errno => e
```

Comment:
`rescue Errno` does not work. Please test. But why are you adding two nested rescues??

Reply:
@mvidner my fault. I am testing it in irb, that it throws Errno::<something> and I do not test that Errno is common base.

Reply:
nested rescue was mistake in approach where are try two possible solutions and does not clean it enough.